frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -864,6 +869,19 @@ export function ContainersSection(props: { resource: KubeObjectInterface | null
         )}
       </SectionBox>
 
+      {ephermeralContainers && (
```

Comment:
This doesn't have the same behavior as the init containers when since it will show the section even when the ephemeral containers list is empty.

Reply:
Fixed using ephemeralContainers.length